conf/routes

```diff
@@ -1,5 +1,6 @@
 GET     /                                   controllers.MainController.index
 GET     /users                              controllers.UserController.getAllUsers
+POST    /users/search                       controllers.UserController.searchUsers
```

Comment:
mm
I think it should be a GET operation, because we are not modifying anything, we're just getting from the db.

With the current implementation it needs to be a POST, but we will need to rethink it